packages/session-replay-browser/src/config/joined-config.ts

```diff
@@ -29,7 +27,15 @@ export class SessionReplayJoinedConfigGenerator {
     // Special case here as optOut is implemented via getter/setter
     config.optOut = this.localConfig.optOut;
     try {
-      const samplingConfig = await this.remoteConfigFetch.getSamplingConfig(sessionId);
+      const sessionReplayConfig =
+        this.remoteConfigFetch &&
+        ((await this.remoteConfigFetch.getRemoteConfig(
+          'sessionReplay',
+          'sr_sampling_config',
+          sessionId,
+        )) as SessionReplayRemoteConfig['sr_sampling_config']);
+
+      const samplingConfig = sessionReplayConfig;
```

Comment:
Could we just use the one variable?

```suggestion
      const samplingConfig =
        this.remoteConfigFetch &&
        ((await this.remoteConfigFetch.getRemoteConfig(
          'sessionReplay',
          'sr_sampling_config',
          sessionId,
        )) as SessionReplayRemoteConfig['sr_sampling_config']);
```

Reply:
Whoops yes